pimd/pim_register.c

```diff
@@ -113,11 +113,36 @@ void pim_register_stop_send(struct interface *ifp, struct prefix_sg *sg,
 	++pinfo->pim_ifstat_reg_stop_send;
 }
 
+static void pim_reg_stop_upstream(struct pim_instance *pim,
+				  struct pim_upstream *up)
+{
+	switch (up->reg_state) {
+	case PIM_REG_NOINFO:
+	case PIM_REG_PRUNE:
+		return;
+		break;
```

Comment:
Can we omit this, or does omitting it generate compiler warnings? Ditto L134

Reply:
May be compiler thrown some warning. I dont remember why that was put. Will try removing and check once. If there were no warnings, will update in the next push after modifying the if condition for upstream.

Reply:
@qlyoung Removed the break and left to return. No warning shown in my compilation.